src/number_theory/bernoulli.lean

```diff
@@ -258,3 +267,34 @@ begin
   { ext x, rw bernoulli_eq_bernoulli' (succ_ne_zero x ∘ succ.inj) },
   { ring },
 end
+
+theorem bernoulli_power_series :
+  power_series.mk (λ n, (bernoulli n / n! : ℚ)) * (exp ℚ - 1) = X :=
+begin
+  suffices f : eval_neg_hom (power_series.mk (λ n, (bernoulli n / n! : ℚ)) *
+    (exp ℚ - 1)) = eval_neg_hom X,
+  { suffices g : function.injective eval_neg_hom,
+    { rwa g.eq_iff at f, },
+    apply rescale_injective,
+    norm_num, },
+  simp only [map_one, map_mul, eval_neg_hom_X, map_sub],
+  suffices h :
```

Comment:
The `simp` can be tucked into the `suffices` by replacing `apply mul_right_cancel' hexp h` with `simpa using mul_right_cancel' hexp h` on line 291 below.

Reply:
done